src/physlr-overlap.cc

```diff
@@ -191,94 +192,102 @@ main(int argc, char* argv[])
 		std::ifstream fh;
 		fh.open(itr->c_str());
 		std::string line;
-		std::cerr << "Loading file" << *itr << std::endl;
+		std::cerr << "Loading file " << *itr << std::endl;
 		while (getline(fh, line)) {
 			std::stringstream ss(line);
 			ss >> barcodeBuffer;
 			const auto& barcode = barcodes.find(barcodeBuffer);
 			if (barcode == barcodes.end()) {
 				barcodeToStr.emplace_back(barcodeBuffer);
 				barcodes[barcodeBuffer] = barcodeToStr.size() - 1;
+				barcodeToMinimizer.emplace_back(std::vector<Minimizer>());
 				while (ss >> minimizerBuffer) {
+					barcodeToMinimizer[barcodeToStr.size() - 1].emplace_back(minimizerBuffer);
 					minimizerToBarcode[minimizerBuffer].insert(barcodeToStr.size() - 1);
 				}
 			} else {
 				while (ss >> minimizerBuffer) {
+					barcodeToMinimizer[barcode->second].emplace_back(minimizerBuffer);
 					minimizerToBarcode[minimizerBuffer].insert(barcode->second);
 				}
 			}
 		}
 	}
 
 #if _OPENMP
-	std::cerr << "Finished constructing minimizerToBarcodes in sec: " << omp_get_wtime() - sTime
-	          << std::endl;
+	std::cerr << "Finished constructing barcodeToMinimizer and minimizerToBarcode in sec: "
+	          << omp_get_wtime() - sTime << std::endl;
 	sTime = omp_get_wtime();
 #endif
 	std::cerr << "Memory usage: " << double(memory_usage()) / double(1048576) << "GB" << std::endl;
 
+	std::cout << "U\tn\n";
+	std::string bufferString;
+	// print out vertexes + counts
+	for (const auto& itr : barcodes) {
+		bufferString.clear();
+		bufferString += itr.first;
+		bufferString += "\t";
+		bufferString += std::to_string(barcodeToMinimizer[itr.second].size());
+		bufferString += "\n";
+		std::cout << bufferString;
+	}
+	std::cout << "\nU\tV\tn" << std::endl;
```

Comment:
Are we still calling this `n` in the code, or should we commit to the `m` change?

Reply:
I think if the file uses `m` in the name then it is good enough. If change it here, I will also have to chage the python read graph code

Reply:
@schutzekatze - What do you think?? I think it's a bit confusing to have 2 names for the parameter... A user couldn't really understand the columns in that file if they wanted to?

I'd prefer just make the full change to `m` to avoid confusing future maintainers of the code/users, but that's just my opinion

Reply:
If we do decide to change it, it will be on another PR. Do you have any other comments. If not i will merge this to master